packages/teams-js/src/public/app.ts

```diff
@@ -589,9 +588,14 @@ export namespace app {
                   if (!isNaN(compareSDKVersions(runtimeConfig, defaultSDKVersionForCompatCheck))) {
                     GlobalVars.clientSupportedSDKVersion = runtimeConfig;
                   }
-                  /* eslint-disable-next-line strict-null-checks/all */ /* Fix tracked by 5730662 */
-                  const givenRuntimeConfig: IRuntime = JSON.parse(clientSupportedSDKVersion);
-                  clientSupportedSDKVersion && applyRuntimeConfig(givenRuntimeConfig);
+                  const givenRuntimeConfig: IRuntime | null = JSON.parse(clientSupportedSDKVersion);
+                  initializeHelperLogger('givenRuntimeConfig parsed to %o', givenRuntimeConfig ?? 'null');
+
+                  if (!givenRuntimeConfig) {
+                    throw new Error('givenRuntimeConfig successfully parsed to null');
```

Comment:
This seems like a slightly weird error message containing the word "successfully". Suggest removing the word "successfully".

Reply:
Interesting; I actually intentionally used that word to indicate that the parsing was successful and it was not the result of a parsing error. 🤔

Reply:
I wondered about that perspective and since I don't have a better suggestion, I guess it's fine.

Reply:
Maybe this new string is clearer?